packages/@dcl/inspector/src/components/EntityInspector/TransformInspector/utils.ts

```diff
@@ -1,38 +1,38 @@
 import { TransformType } from '@dcl/ecs'
-import { Quaternion } from '@dcl/ecs-math'
+import { Quaternion } from '@babylonjs/core'
```

Comment:
I haad to change the implementation to use babylon's Quaternion instead of the `@dcl/ecs-math` because I could not make jest work with that package :(